### PR TITLE
Normalize code block indicators in README

### PR DESCRIPTION
**Pull request by waldyrious**
- Remove space before the two colons, since most instances of the syntax used this format
- Add missing `::` before the code block for the `-S`/`--skip` option

### CODE REVIEW

**Comment by DimitriPapadopoulos** [comment]
The space before `::` is not just a question of format:
* `foo. ::` → `foo.`
* `foo::` → `foo:`
* `foo.::` → `foo.:`

We do not want `.::` without a space, could you please change everything back to `. ::` with a space, or `::` without a point.

File: README.rst [hunk 1]
```diff
@@ -42,15 +42,15 @@ The ``-w`` flag will actually implement the changes recommended by codespell. No
```
Comment (DimitriPapadopoulos) [inline README.rst:42]: The `.::` above needs to be fixed.

File: README.rst [hunk 2]
```diff
@@ -42,15 +42,15 @@ The ``-w`` flag will actually implement the changes recommended by codespell. No
 
     codespell -I FILE, --ignore-words=FILE
 
-The ``-I`` flag can be used for a list of certain words to allow that are in the codespell dictionaries. The format of the file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said list of allowed words. **Important note:** The list passed to ``-I`` is case-sensitive based on how it is listed in the codespell dictionaries. ::
+The ``-I`` flag can be used for a list of certain words to allow that are in the codespell dictionaries. The format of the file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said list of allowed words. **Important note:** The list passed to ``-I`` is case-sensitive based on how it is listed in the codespell dictionaries.::
```
Comment (DimitriPapadopoulos) [inline README.rst:46]: Either `. ::` or `::`.

File: README.rst [hunk 3]
```diff
@@ -42,15 +42,15 @@ The ``-w`` flag will actually implement the changes recommended by codespell. No
...
 
     codespell -L word1,word2,word3,word4
 
-The ``-L`` flag can be used to allow certain words that are comma-separated placed immediately after it.  **Important note:** The list passed to ``-L`` is case-sensitive based on how it is listed in the codespell dictionaries. ::
+The ``-L`` flag can be used to allow certain words that are comma-separated placed immediately after it.  **Important note:** The list passed to ``-L`` is case-sensitive based on how it is listed in the codespell dictionaries.::
```
Comment (DimitriPapadopoulos) [inline README.rst:50]: Revert to `. ::`

File: README.rst [hunk 4]
```diff
@@ -42,15 +42,15 @@ The ``-w`` flag will actually implement the changes recommended by codespell. No
...
 
     codespell -x FILE, --exclude-file=FILE
 
-Ignore whole lines that match those in ``FILE``.  The lines in ``FILE`` should match the to-be-excluded lines exactly.
+Ignore whole lines that match those in ``FILE``.  The lines in ``FILE`` should match the to-be-excluded lines exactly.::
```
Comment (DimitriPapadopoulos) [inline README.rst:54]: Change to `. ::` with a space

File: README.rst [hunk 5]
```diff
@@ -66,7 +66,7 @@ Useful commands::
     codespell -d -q 3 --skip="*.po,*.ts,./src/3rdParty,./src/Test"
 
 List all typos found except translation files and some directories.
-Display them without terminal colors and with a quiet level of 3. ::
```
Comment (DimitriPapadopoulos) [inline README.rst:69]: Revert to `. ::`

**Review by DimitriPapadopoulos — CHANGES_REQUESTED** [review]

**Comment by waldyrious** [comment]
Thanks for the insightful comments, @DimitriPapadopoulos! I decided to use `::` at the end of the line when the code block is related to that line, and to place the `::` in its own line when the code block is unrelated to the previous line (in several cases they are related to the line below).

I've pushed a new commit with these changes. Please take a look and let me know what you think!

**Comment by DimitriPapadopoulos** [comment]
Looks good, more readable and easier to manitain this way. Thank you @waldyrious!

**Review by DimitriPapadopoulos — APPROVED** [review]

File: README.rst [hunk 6]
```diff
@@ -34,24 +34,34 @@ Usage
 
 For more in depth info please check usage with ``codespell -h``.
 
-Some noteworthy flags::
+Some noteworthy flags:
+
+::
```
Comment (jdufresne) [inline README.rst:39]: Just a thought, but for code highlighting, could specify the parser:

```suggestion
.. code-block:: sh
```

Comment (waldyrious) [reply README.rst:39]: Great idea! I'll do that.

Comment (waldyrious) [reply README.rst]: Done with c0f049aa.

**Comment by waldyrious** [comment]
For convenience, [here's the rendered version](https://github.com/waldyrious/codespell/blob/readme-normalize-code-blocks/README.rst).

**Review by jdufresne — APPROVED** [review]

**Comment by jdufresne** [comment]
From the GitHub actions:

> warning: check: Cannot analyze code. Pygments package not found. (line 30)

Might need to include pygments with the `[dev]` group to get the highlighting:

https://pypi.org/project/Pygments/

**Comment by waldyrious** [comment]
> Might need to include pygments with the `[dev]` group to get the highlighting

How do I do that? Something related to [this line](https://github.com/codespell-project/codespell/blob/28ea98b770b3d664a79d269c0f821f0a1ed2ff46/.github/workflows/codespell-private.yml#L39) in the `codespell-private.yml` GitHub actions definition file? Would it be sufficient to add `Pygments` to [this list](https://github.com/codespell-project/codespell/blob/28ea98b770b3d664a79d269c0f821f0a1ed2ff46/pyproject.toml#L34-L41) in `pyproject.toml`?

**Comment by waldyrious** [comment]
> Would it be sufficient to add `Pygments` to [this list](https://github.com/codespell-project/codespell/blob/28ea98b770b3d664a79d269c0f821f0a1ed2ff46/pyproject.toml#L34-L41) in `pyproject.toml`?

Looks like it was :) I did that in 80a611a.

**Comment by larsoner** [comment]
Thanks @waldyrious !

**Pull request merged**